radsat-sk/operation/crypt/encrypt.c

```diff
@@ -0,0 +1,58 @@
+#include "encrypt.h"
+
+int getPaddedBufSize(int plaintextLen);
+void padBuffer(uint8_t* buff, uint8_t* paddedBuff, int buffLen, int newBuffLen);
+void encrypt(uint8_t* plaintext, int plaintextLen);
+
+
+/**
+ * @brief Gets the next multiple of 16 from the size of the input buffer
+ * @header "comms-processing/crypt/encrypt.h"
+ * @param plaintextLen - The size of the unpadded buffer containing data to be encrypted.
+ * @pre need to have a variable storing the size of the plaintext buffer
+ * @return The size of the new buffer to be created
+ */
+int getPaddedBufSize(int plaintextLen){
```

Comment:
Make sure to add spaces around your curly bois. Also, if an if-block just goes to a return statement, you don't need an else block.

Also, newBufSize uses abreviations - against the coding standard. Also, either define the variable at the top of the function, or don't define one at all (just directly return plaintextLen + MODULUS, for example)